tutorials/tour/unified-types-diagram.dot

```diff
@@ -0,0 +1,13 @@
+digraph UnifiedTypes {
```

Comment:
I think the graph is still useful to give users a global picture of what is written in this page. But I agree with point 1. of @Ichoran.